checkov/arm/runner.py

```diff
@@ -138,6 +140,7 @@ def add_python_check_results(self, report: Report, runner_filter: RunnerFilter,
                 path_to_convert = (os.path.join(root_folder, arm_file)) if root_folder else arm_file
 
             file_abs_path = os.path.abspath(path_to_convert)
+            cleaned_path = clean_file_path(Path(arm_file))
```

Comment:
@omriyoffe-panw could you also add a test to check this use-case?

Reply:
I removed this since the python checks are not relevant for the custom policies preview flow